doc/changelog/03-notations/11120-master+refactoring-application-printing.rst

```diff
@@ -0,0 +1,16 @@
+- **Fixed:** Inheritance of implicit arguments across notations made
+  uniform in parsing and printing. To the exception of notations of
+  the form ``Notation @string := @foo`` and ``Notation @ident := @@qualid`` which
```

Comment:
Sorry I was a bit sloppy in my examples, you additionally need to replace the double backticks by simple backticks preceded by `:n:`. E.g.:
```suggestion
  the form :n:`Notation @string := @qualid` and :n:`Notation @ident := @@qualid` which
```

Reply:
OK, I made a change, please check. In particular, is my ``n:`@@qualid` `` correct?

Reply:
I think so yes. However, what is not is the `@foo` which is interpreted as a token `foo` which does not exist, which breaks the build. If you absolutely want to keep `foo` here, and not use a relevant token like `qualid`, then we need to escape the `@`. I am not sure how to do this, though.